Lvl99GameJam/Assets/Scripts/UI Scripts/GridSpaceInteractions.cs

```diff
@@ -81,6 +91,28 @@ private void DestroyPlacementIfNecessary()
         }
     }
 
+    private void ShowSpritePreviewIfNecessary()
+    {
+        if (!dummyPlacement)
+        {
+            dummyPlacement = Instantiate(dummyObject, transform.position, Quaternion.identity);
+            if (placementPrefab == elevatorPrefab)
+            {
+                dummyPlacement.GetComponent<SpriteRenderer>().sprite = elevatorSprite;
+            }
+            else
+            {
+                dummyPlacement.GetComponent<SpriteRenderer>().sprite = fanSprite;
+            }
+        }
+    }
+
+    private void DestroyPreview()
```

Comment:
Might still have to be a conditional destruction if we decided not to show preview because there was an existing placement I think.